recipes/cairo/all/conanfile.py

```diff
@@ -99,47 +119,73 @@ def build(self):
     def _build_msvc(self):
         with tools.chdir(self._source_subfolder):
             # https://cairographics.org/end_to_end_build_for_win32/
-            win32_common = os.path.join('build', 'Makefile.win32.common')
-            tools.replace_in_file(win32_common, '-MD ', '-%s ' % self.settings.compiler.runtime)
-            tools.replace_in_file(win32_common, '-MDd ', '-%s ' % self.settings.compiler.runtime)
-            tools.replace_in_file(win32_common, '$(ZLIB_PATH)/lib/zlib1.lib',
-                                                self.deps_cpp_info['zlib'].libs[0] + '.lib')
-            tools.replace_in_file(win32_common, '$(LIBPNG_PATH)/lib/libpng16.lib',
-                                                self.deps_cpp_info['libpng'].libs[0] + '.lib')
-            tools.replace_in_file(win32_common, '$(FREETYPE_PATH)/lib/freetype.lib',
-                                                self.deps_cpp_info['freetype'].libs[0] + '.lib')
+            win32_common = os.path.join("build", "Makefile.win32.common")
+            tools.replace_in_file(
+                win32_common, "-MD ", "-%s " % self.settings.compiler.runtime
+            )
+            tools.replace_in_file(
+                win32_common, "-MDd ", "-%s " % self.settings.compiler.runtime
+            )
+            tools.replace_in_file(
+                win32_common,
+                "$(ZLIB_PATH)/lib/zlib1.lib",
+                self.deps_cpp_info["zlib"].libs[0] + ".lib",
+            )
+            tools.replace_in_file(
+                win32_common,
+                "$(LIBPNG_PATH)/lib/libpng16.lib",
+                self.deps_cpp_info["libpng"].libs[0] + ".lib",
+            )
+            tools.replace_in_file(
+                win32_common,
+                "$(FREETYPE_PATH)/lib/freetype.lib",
+                self.deps_cpp_info["freetype"].libs[0] + ".lib",
+            )
             with tools.vcvars(self.settings):
                 env_msvc = VisualStudioBuildEnvironment(self)
-                env_msvc.flags.append('/FS')  # C1041 if multiple CL.EXE write to the same .PDB file, please use /FS
+                env_msvc.flags.append(
+                    "/FS"
+                )  # C1041 if multiple CL.EXE write to the same .PDB file, please use /FS
```

Comment:
Not sure that all these formatting modifications are needed. Actually I find them quite cumbersome :s

Reply:
+10, it's hard to review (for little net value), and it makes git blame close to useless.

Reply:
I used https://pypi.org/project/black/ to format the recipe, I'll see if I can change the way I have it setup to only format changed lines.

Reply:
It looks like that is possible by using https://github.com/akaihola/darker I'll revert the formatting commits and then squash my commits before using that, it should then only format the lines I changed.